husarion_ugv_gazebo/launch/simulate_robot.launch.py

```diff
@@ -107,7 +107,6 @@ def generate_launch_description():
             )
         ),
         launch_arguments={
-            "add_wheel_joints": "False",
```

Comment:
This should not be removed.

Reply:
This parameter does not appear any more.